bentoml/__init__.py

```diff
@@ -1,3 +1,4 @@
+# pylint: skip-file
```

Comment:
Why are we adding this now?

Reply:
otherwise pep8 bot would just throw warning.

Reply:
Oh, is there not a pep8-specific one?

Reply:
pep8 just follow pylint warning iirc.